Source/AtomicEditor/Editors/SceneEditor3D/SceneEditor3D.cpp

```diff
@@ -217,6 +226,20 @@ void SceneEditor3D::HandleUpdate(StringHash eventType, VariantMap& eventData)
 {
     if (!cubemapRenderCount_)
         gizmo3D_->Update();
+
```

Comment:
Does this need to be run every frame?  Setting up text on widget controls is seldom a cheap operation